docs/source/learn/eager.md

```diff
@@ -1,14 +1,14 @@
-# Eager Mode + Compile API
+# PyTorch/XLA Compilation: Eager Mode vs. Compile API
 
-In this doc we will go over how to use PyTorch/XLA's new experimental
-`eager` mode with the `compile` API. The goal is to make PyTorch/XLA
-experience more aligned with the native PyTorch and make development
-process easier.
+## Overview
 
-Currently PyTorch/XLA runs on the LazyTensor tracing mode by default. In
-the following code
+PyTorch/XLA integrates PyTorch with the XLA compiler to optimize deep learning
+workloads across various hardware accelerators. Currently PyTorch/XLA runs on
+the LazyTensor tracing mode by default where operations were recorded into a
```

Comment:
... the LazyTensor tracing mode by default where operations are recorded into a ...

Reply:
done